app/tag-management/all-tags/tag-list/tag-list.component.html

```diff
@@ -0,0 +1,22 @@
+<ol class="tag-list">
+  <li *ngFor="let tag of tags; let isLast = last">
+    <div class="tag-header">
+      <h3>{{ tag.name }}</h3>
+      <div *ngIf="tagsEditable" class="tag-actions">
+        <a [routerLink]="['/tags/edit', tag.id]"><md-icon title="edit tag">edit</md-icon></a>
+        <a [routerLink]="['/tags/delete', tag.id]"><md-icon title="delete tag">delete</md-icon></a>
+        <a [routerLink]="['/tags/add-sub-tag', tag.id]"><i class="material-icons">add</i></a>
+        <a [routerLink]="['/tags/remove-sub-tag', tag.id]"><i class="material-icons">clear</i></a>
```

Comment:
I am not sure if I understood this correctly. Do you want the option for deleting a Tag to be presented only if a tag is displayed as subtag of some other tag?